{
	"id": "CVE-2019-14233",
	"vulnerabilities": [
		{
			"id": "CVE-2019-14233",
			"description": "An issue was discovered in Django 1.11.x before 1.11.23, 2.1.x before 2.1.11, and 2.2.x before 2.2.4. Due to the behaviour of the underlying HTMLParser, django.utils.html.strip_tags would be extremely slow to evaluate certain inputs containing large sequences of nested incomplete HTML entities.",
			"severity": [
				{
					"type": "cvss_v30",
					"source": "security.alpinelinux.org",
					"cvss_v30": {
						"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"base_score": 7.5,
						"base_severity": "HIGH",
						"temporal_score": 7.5,
						"temporal_severity": "HIGH",
						"environmental_score": 7.5,
						"environmental_severity": "HIGH"
					}
				}
			],
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2019-14233"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://docs.djangoproject.com/en/dev/releases/security/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.gentoo.org/glsa/202004-17"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.netapp.com/advisory/ntap-20190828-0002/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://www.debian.org/security/2019/dsa-4498"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://www.djangoproject.com/weblog/2019/aug/01/security-releases/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "http://lists.opensuse.org/opensuse-security-announce/2019-08/msg00006.html"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "http://lists.opensuse.org/opensuse-security-announce/2019-08/msg00025.html"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://groups.google.com/forum/#%21topic/django-announce/jIoju2-KLDs"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/STVX7X7IDWAH5SKE6MBMY3TEI6ZODBTK/"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://seclists.org/bugtraq/2019/Aug/15"
				}
			],
			"published": "2019-08-02T15:15:11Z",
			"modified": "2024-06-06T12:36:02.052641Z"
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:3.10",
			"vulnerable": true,
			"package": {
				"name": "py-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.23-r0"
					}
				],
				"fixed": [
					"1.11.23-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.11",
			"vulnerable": true,
			"package": {
				"name": "py3-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.23-r0"
					}
				],
				"fixed": [
					"1.11.23-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.12",
			"vulnerable": true,
			"package": {
				"name": "py3-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.23-r0"
					}
				],
				"fixed": [
					"1.11.23-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.7",
			"vulnerable": true,
			"package": {
				"name": "py-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.23-r0"
					}
				],
				"fixed": [
					"1.11.23-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.8",
			"vulnerable": true,
			"package": {
				"name": "py-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.23-r0"
					}
				],
				"fixed": [
					"1.11.23-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.9",
			"vulnerable": true,
			"package": {
				"name": "py-django"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.11.23-r0"
					}
				],
				"fixed": [
					"1.11.23-r0"
				]
			}
		}
	],
	"data_source": "alpine-osv"
}
